native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/utils/ValueTypeUtils.java

```diff
@@ -60,22 +64,31 @@ public static Type getValueTypeStream(String key) {
                 break;
         }
         Record contextStream = new Record();
-        contextStream.addStreamField(key, "content");
+        if (componentsModuleMap.containsKey(key) && protofileModuleMap.containsKey(filename) &&
+                !componentsModuleMap.get(key).equals(protofileModuleMap.get(filename))) {
+            contextStream.addStreamField(componentsModuleMap.get(key).substring(componentsModuleMap.get(key)
+                    .lastIndexOf(PACKAGE_SEPARATOR) + 1) + COLON + key, "content");
+        } else {
+            contextStream.addStreamField(key, "content");
```

Comment:
Let's make "content" a constant since used in multiple places.

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-grpc/pull/885/commits/563e2322c111a937d37eb479bc08eabd020194bd